.storybook/test-runner.ts

```diff
@@ -0,0 +1,34 @@
+import { type TestRunnerConfig, getStoryContext } from '@storybook/test-runner'
+import { checkA11y, configureAxe, injectAxe } from 'axe-playwright'
+
+const config: TestRunnerConfig = {
+  async preVisit(page) {
+    // Inject Axe utilities in the page before the story renders
+    await injectAxe(page)
+  },
+  async postVisit(page, context) {
+    // Get entire context of a story, including parameters, args, argTypes, etc.
+    const storyContext = await getStoryContext(page, context)
+
+    // Do not test a11y for stories that disable a11y
+    if (storyContext.parameters?.a11y?.test === 'todo') {
+      return
+    }
```

Comment:
Docs use `disable` prop, why check `test`?
```
    // Do not run a11y tests on disabled stories.
    if (storyContext.parameters?.a11y?.disable) {
      return;
    }
```

Reply:
I was following this page: https://storybook.js.org/docs/writing-tests/accessibility-testing#configure-accessibility-tests-with-the-test-addon which suggests adding `parameters: { a11y: { test: 'todo' } },`. But the ignored (todo) stories were actually not being ignored when I run a11y tests. Turned out that parts of the docs are referring to storybook v9 (which is in beta now), and parts to v8.

I got some help from storybook devs: https://github.com/storybookjs/storybook/discussions/31368 (let's keep in mind that they are responsive!).

I thought it would be ok to leave those `test: 'todo'` instead of using some `manual: true` or other way. "todo" feels more meaningful :)

Reply:
I hid the "No accessibility violations detected!" with `verbose: false`. The errors would still appear correctly.

Reply:
lets try to upgrade :crossed_fingers: